R/preprocess.R

```diff
@@ -296,30 +360,37 @@ enw_preprocess_data <- function(obs, by = c(), max_delay = 20,
   )
   diff_obs[, group := new_group][, new_group := NULL]
   obs[, old_group := NULL]
+  
+  # separate obs with and without missing reference date
+  reporting_available <- diff_obs[!is.na(reference_date)]
+  reporting_missing <- diff_obs[is.na(reference_date)]
```

Comment:
We could still run into trouble if the data provided by the user starts at a cumulative count that is nonzero. Should we create another issue for this?

Reply:
yes I think so.